src/main/java/seedu/address/logic/parser/ViewCommandParser.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ClientViewCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ClientViewCommand object
+ */
+public class ViewCommandParser implements Parser<ClientViewCommand> {
```

Comment:
Could you change the class and file name to `ClientViewCommandParser`? I will be updating the ones for Find and Delete in a future PR